### ISSUE TITLE
I can't reproduce your BulkRead example. 

### ISSUE THREAD

**Issue by nikriaz** [borisdj/EFCore.BulkExtensions#518]
I can't reproduce your BulkRead example. 
```
// USE
var items = itemsNames.Select(a => new Item { Name = a }); // creating list of Items where only Name is set
var bulkConfig = new BulkConfig { UpdateByProperties = new List<string> { nameof(Item.Name) } };
context.BulkRead(items, bulkConfig); // Items list will be loaded from Db with data(other properties)
```
In fact, `items` you create is not a list but IEnumerable and this example produces compile compiler Error CS0411:
The type arguments for method 'DbContext.BulkExtensions.BulkRead' cannot be inferred from the usage.

If I convert `items `from an IEnumerable to a List by adding .ToList() in the first line of your example then compile error goes away but getting runtime error instead "The method or operation is not implemented" with no clues what is about.

**Comment by borisdj** [comment]
Yes it should be list:
`var items = itemsNames.Select(a => new Item { Name = a }).ToList();`
I've updated ReadMe.

As for the second error, in source code in EFCoreBulkTest there is example that works.
However if you are using Sqlite, BulkRead for Lite is not yet implemented, probably why you have this exception.

**Comment by nikriaz** [comment]
Thanks! This is really pity. Looking forward for SQLite implementation!

**Comment by borisdj** [comment]
BulkRead is now implemented for Sqlite, v5.0.8.+

**Comment by nikriaz** [comment]
Oh, wow. Never seen that feature request would got implemented so fast. Thanks!
